rusk-wallet/Makefile

```diff
@@ -10,7 +10,9 @@ install: build ## Install the wallet
 clippy: 
 	cargo clippy --all-features --release -- -D warnings
 
-test: build ## Run wallet tests
-	cargo test --release
+test: ## Run wallet tests
+	$(MAKE) -C .. keys
+	$(MAKE) -C .. wasm
```

Comment:
Why do we need this?

Reply:
Aren't they needed to run the node?

Reply:
Make wasm is not needed to run a node anymore